### DISCUSSION TITLE
Odfgrandorgue samplesets

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#956] [General]
Perhaps this post is a little bit OT…
On French website odfgrandorgue can be downloaded many samplesets, some of them very interesting. They all come in exe format and cannot been opened with known archivers like winrar or 7zip but must be executed as windows application. 
maybe any of you knows if there is a way to manage and unpack those files without executing. Thanks in advance for any hints.

**Comment by rousseldenis** [comment]
> Perhaps this post is a little bit OT… On French website odfgrandorgue can be downloaded many samplesets, some of them very interesting. They all come in exe format and cannot been opened with known archivers like winrar or 7zip but must be executed as windows application. maybe any of you knows if there is a way to manage and unpack those files without executing. Thanks in advance for any hints.

You can download the Linux ones. They are in zip format.

**Reply by larspalo** [reply]
I'd be a bit careful about those samplesets... The "producer" has had a bit of a history of not being quite honest in the past. Read https://sourceforge.net/p/ourorgan/mailman/message/31647660/ and follow ups.

**Reply by paolo2504** [reply]
thanks Roussel honestly didnt think at this. i'll give a try.

**Reply by paolo2504** [reply]
Lars, appreciate you sharing this. It's a must to observe and respect copyright laws, primary with the Author's interest. I see your post on sourceforge and Martin's next reply are dated 2013, that is long time ago. I'm curious to know if any reply by the website owner involved during this time.
I'll pay much attention. tks.
